DBusInterface.md

```diff
@@ -30,6 +32,9 @@ The **org.freedesktop.ratbag1.Device** interface provides:
 - Methods:
   - GetProfileByIndex(uint) -> returns the object path for the given index
   - Commit() -> commit the changes to the device
+  - GetSvg(string) -> returns the svg for the theme given, must be one of
```

Comment:
Fixed locally, but fwiw I have some plans to extend the dbus documentation with better markup anyway.